### PR TITLE
add jack-o for ggxrd-r2

### PR DESCRIPTION
**Pull request by komidore64**
Not particularly confident in this movelist. Haven't played with Jack-O either

### CODE REVIEW

**Comment by komidore64** [comment]
Jack-O is the first I've run into that has named normals, so that's probably going to break out character pages.

File: src/data/guilty_gear_xrd_rev_2/jack-o.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        - 6D
+        - j.6D
+    - 2P
+    - 2K
+    - 2S
+    - 2H
+    - name: Zest
+      inputs:
+        - 2D
+        - j.2D
```
Comment (yithian) [inline src/data/guilty_gear_xrd_rev_2/jack-o.yml]: Is this redundant with Air Zest (j.2D) ?

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/jack-o.yml]: Buuuuuuh weird. Dustloop shows **air ok** on Zest, but also lists an Air Zest move.

Comment (komidore64) [reply src/data/guilty_gear_xrd_rev_2/jack-o.yml]: I'll remove the redudancy here though.

**Comment by komidore64** [comment]
updated

**Pull request merged**